src/libraries/Microsoft.NETCore.Platforms/readme.md

```diff
@@ -100,7 +100,7 @@ For example:
 
 This will create a new RID for `myLinuxDistro` where `myLinuxDistro` should be the string used for the `ID=` value in the `/etc/os-release` file.
 
-Whenever modifying the `runtimeGroups.props` you should rebuild the project with `/p:UpdateRuntimeFiles=true` so that your changes will be regenerated in the checked-in `runtime.json`.
+Whenever modifying the `runtimeGroups.props` you should rebuild the project with `/t:UpdateRuntimeJson /p:UpdateRuntimeFiles=true` so that your changes will be regenerated in the checked-in `runtime.json`.
```

Comment:
Now that we have a fairly an easy way to produce the package (building the Platforms packages doesn't even require any up-front build), I would recommend to create the package and diff its inputs. The `UpdateRuntimeJson` target is already hooked to run before packaging, so invoking that manually shouldn't be necessary anymore.

```suggestion
Whenever modifying the `runtimeGroups.props` make sure to pack the project via the `dotnet pack` command and inspect if the generated package contains the desired changes.
```